### PR TITLE
PA-2420 - Default to land on inventory list

### PR DESCRIPTION
**Pull request by jdtoombs**
Inventory list view now functions the same as before (defaulting to land), and `PropertyType` is only set to undefined within the query to download the excel/csv report

### CODE REVIEW

File: frontend/src/features/properties/list/PropertyListView.tsx [hunk 1]
```diff
@@ -45,7 +45,7 @@ const initialQuery: IPropertyQueryParams = {
 };
 
 const defaultFilterValues: IPropertyFilter = {
-  searchBy: 'address',
+  searchBy: 'name',
```
Comment (jdtoombs) [inline frontend/src/features/properties/list/PropertyListView.tsx:48]: just to keep consistent with filter elsewhere

File: frontend/src/features/properties/list/PropertyListView.tsx [hunk 2]
```diff
@@ -252,10 +248,10 @@ const PropertyListView: React.FC = () => {
   const dispatch = useDispatch();
 
   const fetch = (accept: 'csv' | 'excel') => {
-    const query = getServerQuery(true, { pageIndex, pageSize, filter, agencyIds });
+    const query = getServerQuery({ pageIndex, pageSize, filter, agencyIds });
     return dispatch(
       download({
-        url: getPropertyReportUrl({ ...query, all: true }),
+        url: getPropertyReportUrl({ ...query, all: true, propertyType: undefined }),
```
Comment (agalbrai) [inline frontend/src/features/properties/list/PropertyListView.tsx:254]: So this puts both property types into the excel export?

Comment (jdtoombs) [reply frontend/src/features/properties/list/PropertyListView.tsx:254]: Yup, same idea as the previous `ignorePropertyType` but this one only affect the excel/csv reports allowing the normal list view to default to land again

Comment (agalbrai) [reply frontend/src/features/properties/list/PropertyListView.tsx:254]: perfect 👍

**Review by agalbrai — APPROVED** [review]

File: frontend/src/features/properties/list/PropertyListView.tsx [hunk 3]
```diff
@@ -66,15 +66,12 @@ const defaultFilterValues: IPropertyFilter = {
  * @param ignorePropType - Whether to ignore the PropertyTypes (for doing excel export with all values)
```
Comment (devinleighsmith) [inline frontend/src/features/properties/list/PropertyListView.tsx:66]: Probably can remove this comment then.

Comment (devinleighsmith) [reply frontend/src/features/properties/list/PropertyListView.tsx:66]: we don't need the ignorePropertyType anymore?

**Review by devinleighsmith — APPROVED** [review]

**Pull request merged**